libnopegl/src/node_text.c

```diff
@@ -81,12 +81,14 @@ struct text_opts {
 };
 
 struct text_priv {
+    /* characters (fg) */
```

Comment:
nit: no need to specify (fg), imo.

Reply:
Okay, dropped (I wanted to make a distinction between the fg and bg pipelines)